README.md

```diff
@@ -21,6 +21,7 @@ project.
 ## Requirements
 
 - JupyterLab >= 2.0
+- Python 64-bit Version
```

Comment:
Since we are adding to the README background on Python installation, can you also add what Python *versions* are supported? You can see from https://github.com/ballet/ballet-assemble/blob/master/setup.py#L15 which Python versions are accepted by the build system, and can copy this over to the README.